src/commands/api-mesh/status.js

```diff
@@ -35,7 +42,47 @@ class StatusCommand extends Command {
 
 		if (meshId) {
 			try {
+				const { showCloudflareURL: showEdgeMeshUrl } = await getTenantFeatures(imsOrgCode);
 				const mesh = await getMesh(imsOrgId, projectId, workspaceId, workspaceName, meshId);
+				if (showEdgeMeshUrl) {
+					const meshDeployments = await getMeshDeployments(
+						imsOrgCode,
+						projectId,
+						workspaceId,
+						meshId,
+					);
+					this.log(chalk.blackBright('Edge Mesh Status:'));
+					switch (String(meshDeployments.status).toLowerCase()) {
```

Comment:
Added `String(meshDeployments.status).toLowerCase()` to handle `SUCCESS`, `success` and `Success`. If the entry in table would always be unique(lowercase) we can remove this.